graph_snyk_cve_sync.py

```diff
@@ -418,11 +497,12 @@ def _insert_cves(self):
         dry_run = self.helper.is_dry_run()
         if dry_run:
             logger.info("Dry run mode is on. No ingestion will take place".center(30, '-'))
-        for eco in ('maven', 'npm', 'pypi'):
+        for eco in ('maven', 'npm', 'pypi', 'golang'):
```

Comment:
Can we avoid doing this all over the code and do this check only once

Reply:
Will change this

Reply:
Done

Reply:
I dont see this change, i still it in multiple places.

Reply:
:) not yet pushed. was waiting for your response on the prev comment